globals/macros.yaml

```diff
@@ -21,7 +21,7 @@
       - shell: |
           sshvars=($SSH_CLIENT)
           JENKINS_MASTER_IP=${{sshvars[0]}}
-          JENKINS_SLAVE_IP=$(hostname --ip-address)
+          JENKINS_SLAVE_IP=$(ip route get 1 | awk '{ print $NF; exit }')
```

Comment:
how about to use 8.8.8.8 instead of 1(=1.0.0.0), to make it more clear (from network point of view)?

In addition, we need to mention that TOAD assumes IPv4 network with this code, anywhere...